components/GameCard/GameCard.js

```diff
@@ -1,16 +1,31 @@
 import styled from "styled-components";
-import { BsPersonCircle } from "react-icons/bs";
+import { BsPersonCircle, BsX } from "react-icons/bs";
+import DeleteButton from "../DeleteButton";
 
-export default function GameCard({ type, name, date, time, place, court }) {
+export default function GameCard({
+  type,
+  name,
+  date,
+  time,
+  place,
+  court,
+  deleteCard,
+  id,
+}) {
   return (
     <Card>
-      <CardType>{type}</CardType>
+      <CardType type={type}>{type}</CardType>
       <CardContainer>
         <PlayerDiv>
           <p>
             <BsPersonCircle /> {name}
           </p>
         </PlayerDiv>
+        <DeleteButtonContainer>
+          <DeleteButton>
+            <BsX onClick={() => deleteCard(id)} />
```

Comment:
click event on icon? Set it on the button

Reply:
Hey, unfortunately onClick is not working when set on the deletebutton styled component... tough I did set is on the DeleteButtonContainer.